tunnel/main.mk

```diff
@@ -2,12 +2,12 @@
 ########################################################################################################################
 SSH_CONFIG ?= $(INFRA_DIR)/env/$(ENV)/ssh.config
 
-CMD_BASTION_SSH_TUNNEL_CONFIG = echo $(shell cat $(INFRA_DIR)/env/$(ENV)/output.json | $(JQ) -r '.ssh_forward_config.value[]' > $(SSH_CONFIG)) && echo "\# SSH Tunnel Config \n\# Use the Forward ports to connect to remote instances (localhost:<PORT>)\n-----" && cat $(SSH_CONFIG)
+CMD_BASTION_SSH_TUNNEL_CONFIG = echo $(shell $(AWS) ssm get-parameter --name "output" --with-decryption |$(JQ) -r '.Parameter.Value'| $(BASE64)  -d | $(JQ) -r '.ssh_forward_config.value[]' > $(SSH_CONFIG)) && echo "\# SSH Tunnel Config \n\# Use the Forward ports to connect to remote instances (localhost:<PORT>)\n-----" && cat $(SSH_CONFIG)
```

Comment:
- Maybe let's name it `CMD_BASTION_SSH_TUNNEL_CONFIG_CREATE`? (action is more clear)
- Ensure we update the param name
- Feel free to use a $(HASHSIGN) macros that we have in ICMK , so the IDE formatter doesn't break with `\#`
- Please remove extra spaces that are not nesseary

Reply:
1. Understand
2. What do you mean by param name? Do you mean `--name "output"`?
3. This was in the original icmk, so this part came unchanged. I'll look at how to use here $(HASHSIGN) macros.
4. Understand